web/impact/impact/v1/helpers/criterion_helper.py

```diff
@@ -17,6 +17,13 @@
 
 
 class CriterionHelper(ModelHelper):
+    '''Encapsulates business logic for Criteria, including logic
+    around allocation, analysis, and cloning.
+    The CriterionHelper superclass counts judges without regard to any
+    features. Therefore it is a suitable class to use for the "reads"
+    criterion. 
+    '''
```

Comment:
formatting consistency (having the close comment on the same line)

Reply:
Python  community standards (PEP257) say multiline docstrings should have the closing quotes on the following line. Fixing this in the places where I got it wrong.